pandas/core/arrays/categorical.py

```diff
@@ -1732,17 +1744,25 @@ def fillna(self, value=None, method=None, limit=None):
         return self._constructor(values, categories=self.categories,
                                  ordered=self.ordered, fastpath=True)
 
-    def take_nd(self, indexer, allow_fill=True, fill_value=None):
-        """ Take the codes by the indexer, fill with the fill_value.
+    def take_nd(self, indexer, allow_fill=None, fill_value=None):
+        """
+        Return the indices
```

Comment:
-> `Take elements from the Categorical` ? (in any case, it are not the indices that are returned :-))